src/natcap/invest/sdr/sdr.py

```diff
@@ -154,13 +153,13 @@
 _OUTPUT_BASE_FILES = {
     'rkls_path': 'rkls.tif',
     'sed_export_path': 'sed_export.tif',
-    'sed_retention_index_path': 'sed_retention_index.tif',
-    'sed_retention_path': 'sed_retention.tif',
     'sed_deposition_path': 'sed_deposition.tif',
     'stream_and_drainage_path': 'stream_and_drainage.tif',
     'stream_path': 'stream.tif',
     'usle_path': 'usle.tif',
     'watershed_results_sdr_path': 'watershed_results_sdr.shp',
+    'total_retention_path': 'total_retention.tif',
```

Comment:
Again, inconsistency between PEP naming and code naming. Just want to make sure our source code is aligned with the UG.

Reply:
I've updated this to be more consistent and according to the slack discussion at https://stanford-natcap.slack.com/archives/CECFUU1DW/p1649109737853559?thread_ts=1647968772.949939&cid=CECFUU1DW.